gax/src/main/java/com/google/api/gax/retrying/NoopRetryingContext.java

```diff
@@ -51,4 +53,14 @@ public static RetryingContext create() {
   public ApiTracer getTracer() {
     return NoopApiTracer.getInstance();
   }
+
+  @Override
+  public RetrySettings getRetrySettings() {
+    return null;
+  }
+
+  @Override
+  public Set<Code> getRetryableCodes() {
```

Comment:
empty set instead of null.

Reply:
The contract of the (existing) interface is that the method should return:
* `null` if the default settings should be used.
* An empty set if retries should be disabled.
* A set with codes if the default retryable codes should be overridden.